.github/workflows/django_test.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+name: django-tests
+jobs:
+  system-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.8
+
+      - name: Start YugabyteDB on docker
+        run: docker run -d --name yugabyte -p5437:5433 yugabytedb/yugabyte:2.12.3.0-b19 bin/yugabyted start --daemon=false --tserver_flags='enable_pg_savepoints=true'
+
+      - name: Run Django tests
+        run: sh django_test_suite.sh
+        env: 
+          DJANGO_TEST_APPS: absolute_url_overrides admin_autodiscover admin_checks admin_default_site admin_registration admin_scripts app_loading apps asgi async bash_completion builtin_server cache check_framework conditional_processing constraints context_processors csrf_tests custom_lookups custom_methods datatypes db_typecasts db_utils dbshell.test_postgresql decorators deprecation dispatch empty empty_models field_deconstruction field_defaults field_subclassing file_storage files fixtures_model_package from_db_value handlers httpwrappers humanize_tests i18n invalid_models_tests logging_tests m2o_recursive mail max_lengths messages_tests middleware middleware_exceptions migrate_signals migrations.test_autodetector migrations.test_commands.AppLabelErrorTests migrations.test_loader migrations.test_commands.MakeMigrationsTests migrations.test_commands.OptimizeMigrationTests migrations.test_commands.SquashMigrationsTests migrations.test_exceptions migrations.test_executor migrations.test_graph migrations.test_multidb migrations.test_operations.FieldOperationTests migrations.test_operations.SwappableOperationTests migrations.test_optimizer migrations.test_questioner migrations.test_state migrations.test_writer migration_test_data_persistence migrations2 model_enums model_indexes model_meta model_utils no_models or_lookups pagination project_template properties requests reserved_names resolve_url responses save_delete_hooks servers sessions_tests settings_tests shell shortcuts signed_cookies_tests signing sitemaps_tests sites_tests staticfiles_tests str template_backends template_loader test_client test_client_regress test_exceptions test_runner transaction_hooks transactions urlpatterns urlpatterns_reverse user_commands utils_tests validators version wsgi
```

Comment:
I'd recommend to run all `migrations` and `schema` tests and list tests that don't work in `django_test_expected_failures`/`django_test_skips` with appropriate comments. See how it'd handled in [django-cockroachdb](https://github.com/cockroachdb/django-cockroachdb/blob/46df2730abe8fcdc1e4b0370023eb174b058331f/django_cockroachdb/features.py#L89-L210).

Reply:
I have updated the file now to run both `migrations` and `schema` tests.